zero.feature

```diff
@@ -0,0 +1,36 @@
+  Feature: As a calculator user I want to add, multiply and divide numbers So I can do simple maths quickly
+
+    Scenario: Add two numbers
```

Comment:
Teraz już się czepiam, ale scenariusz nazywa się "Add two numbers", po czym testujesz w nim wszystkie operacje. :) Drugi nazywa się tak samo, a testujesz w nim 3 działania.
Albo rozbijaj takie testy na bardziej "jednostkowe" - jeden scenariusz na każdą operację, albo nadaj im lepsze nazwy - w pierwszym na przykład równie dobrze możesz twierdzić, że testujesz, że każda zmiana operatora nie wyzerowuje pól z wprowadzonymi liczbami ;).
Pamiętaj też testować jakieś szczególne przypadki - tutaj, w kalkulatorze, to dzielenie przez 0 na przykład. :)